modin/pandas/test/test_io.py

```diff
@@ -397,6 +398,23 @@ def test_from_excel():
     teardown_excel_file()
 
 
+def test_from_excel_all_sheets():
+    setup_excel_file(SMALL_ROW_SIZE)
+
+    pandas_df = pandas.read_excel(TEST_EXCEL_FILENAME, sheet_name=None)
+    modin_df = pd.read_excel(TEST_EXCEL_FILENAME, sheet_name=None)
+
+    assert isinstance(pandas_df, OrderedDict)
+    assert isinstance(modin_df, OrderedDict)
+
+    assert pandas_df.keys() == modin_df.keys()
+
+    for key in pandas_df.keys():
+        assert modin_df.get(key).equals(pandas_df.get(key))
```

Comment:
You should be able to use `modin_df_equals_pandas` after the conversion. Each of these is a `pandas` DataFrame currently.